.github/actions/report-job-status/README.md

```diff
@@ -0,0 +1,31 @@
+# Send VK Teams notifications on failures
+
+Action uses[`report-job-status action`](https://github.com/tarantool/actions/tree/master/report-job-status)
+and change the chat for notifications:
+* Main chat (Tarantool CI/CD reports) - if the job started for stable branch
+(`master`, `2.10`, `1.10`)
+* Personal chat, created by committer – for any PR or for any event in other
+branches
+
+## How to create personal chat
+
+1. Create chat in VK Team, make it public (access can be limited by enabling 
+"Join in Approval" in chat settings)
+2. Change URL of the chat: use URL of main VK Teams chat and add
+"_<your_github_nickname>", example: `main_chat_mynickname`
```

Comment:
for example: `https://vkteams.example.com/main_chat_id_mynickname`

Reply:
Done